scripts/Constants.s.sol

```diff
@@ -88,6 +88,9 @@ uint256 constant CHAIN_SOURCE = CHAIN_ETHEREUM;
 uint256 constant BASE_18 = 1e18;
 uint256 constant BASE_9 = 1e9;
 
+uint256 constant BASE_GAS = 100000;
+uint256 constant GAS_MULTIPLIER = 150000; // 1.5x
```

Comment:
I just tried locally and the eth sent seems pretty low (0.001), I am not sure it will be enough for the message. So we can be more conservative and set this value to x3 first and then decrease if we are really too high

Reply:
I have tested on gnosis to polygon and it worked pretty well. Maybe we can increase it but 3x seems a bit too much